### PR TITLE
rewrite rpc tracing

### PR DESCRIPTION
**Pull request by danmanj**

### CODE REVIEW

File: distbench_test_sequencer_test.cc [hunk 1]
```diff
@@ -1194,10 +1206,23 @@ tests {
   ASSERT_NE(dest_rpc, dest->second.rpc_logs().end());
   ASSERT_TRUE(dest_rpc->second.failed_rpc_samples().empty());
   ASSERT_EQ(dest_rpc->second.successful_rpc_samples_size(), 25);
+  int64_t iterations = 0;
```
Comment (oserres) [inline distbench_test_sequencer_test.cc:1209]: Maybe iterations_bits or something like that, to make it easier to understand that it's not just a count.

File: distbench_test_sequencer_test.cc [hunk 2]
```diff
@@ -1221,10 +1245,33 @@ tests {
   ASSERT_NE(dest_rpc, dest->second.rpc_logs().end());
   ASSERT_TRUE(dest_rpc->second.failed_rpc_samples().empty());
   ASSERT_EQ(dest_rpc->second.successful_rpc_samples_size(), 25);
+
+  iterations = 0;
+  int64_t invocations = 0;
```
Comment (oserres) [inline distbench_test_sequencer_test.cc:1250]: Same as above.

**Review by oserres — APPROVED** [review]
LGTM

File: distbench_test_sequencer_test.cc (hunk 2 above)
Comment (danmanj) [reply distbench_test_sequencer_test.cc:1250]: done

**Pull request merged**